R/pkg/R/DataFrame.R

```diff
@@ -3049,8 +3050,8 @@ setMethod("drop",
 #'
 #' @name histogram
 #' @param nbins the number of bins (optional). Default value is 10.
+#' @param col the column (described by character or Column object) to build the histogram from.
```

Comment:
we generally don't say "Column object" - object system in R is a bit different and I think here "S4 class" would make more sense?
but I'd suggest simplifying it to:
"@param col the column as Character string or a Column to build the histogram from"